models/issues/comment.go

```diff
@@ -897,6 +887,23 @@ func updateCommentInfos(ctx context.Context, opts *CreateCommentOptions, comment
 	return UpdateIssueCols(ctx, opts.Issue, "updated_unix")
 }
 
+func checkAttachments(ctx context.Context, opts *CreateCommentOptions, comment *Comment) error {
```

Comment:
It doesn't seem a proper name. It not only "checks", but also "assigns" or "binds" the attachments to a comment.